robot_modeling/DQ_SerialManipulator.m

```diff
@@ -769,6 +644,10 @@ function update_robot(robot, q)
         for i=1:robot.n_links
             % Since the maximum reaching distance are given by the link offset 
             % and link length, we add them.
+
+            % TODO
+            % This part is strange because it assumes we are using the 
```

Comment:
Change the comment to: "This part of the code assumes we are using (...)"

Reply:
- [ ] Implemented.